build_linux.sh

```diff
@@ -1,14 +1,17 @@
 #!/bin/sh
 set -eu
 
+IMAGE_NAME="tor-brave"
+DOCKERFILE="Dockerfile-linux"
+
 cleanup () {
   echo "cleaning up docker containers/images"
-  docker rm -f tor-brave || true
-  docker rmi tor-brave || true
+  docker rm -f $IMAGE_NAME || true
+  docker rmi $IMAGE_NAME || true
 }
 
 cleanup
-docker build --no-cache -t tor-brave -f Dockerfile-linux \
+docker build --no-cache -t $IMAGE_NAME -f $DOCKERFILE \
```

Comment:
There's multiple places in this PR where variables are not double-quoted and consequently subject to globbing and word splitting. In this case, we can see the assignment a few lines above and these strings happen to be safe, but I'd stick to the rule that every variable needs to be quoted.